### PR TITLE
#163728706: ch-modifiy the README file

### PR DESCRIPTION
**Pull request by kisakyegordon**
#### What does this PR do?
- Modifies the `README` file.

#### Description of Task to be completed?
- Adds details during the setup of `WIRE` on development, to make the process seamless and be  blocker free. 

#### How should this be manually tested?
- Read the `README` file.

#### What are the relevant pivotal tracker stories?
[#163728706](https://www.pivotaltracker.com/story/show/163728706)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,14 +40,16 @@ Series of steps to setup development environment
 * You will need to add the following to your `.env` file:
 ```
 NODE_ENV=development
-API_URL=http://wire.andela.com:8080/api
-ANDELA_API_BASE_URL=https://api-prod.andela.com
+API_URL=http://localhost:3000/api
+ANDELA_API_BASE_URL=https://api-staging.andela.com
```
Comment (brandeddavid) [inline README.md]: Any reason for the change here from prod to staging?

Comment (kisakyegordon) [reply README.md]: - For `Local Development`, its better to make changes to the `staging API`.

Comment (brandeddavid) [reply README.md]: Great if there are instances we are making changes to the Andela API, that makes sense. I thought we were just consuming from the Andela API in which case prod would be the best env to consume from.

**Review by brandeddavid — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kisakyegordon) [reply README.md]: @brandeddavid 
- I see your point, for the sake of `consuming` prod data and not `making changes`, i will revert to  `prod`.

**Pull request merged**